spec.bs

```diff
@@ -954,8 +1104,45 @@ The output of running a FLEDGE generateBid() script, which needs to be scored by
 : <dfn>bid</dfn>
 :: A {{double}}. If the bid is zero or negative, then this interest group will not participate in
   the auction.
-: <dfn>render</dfn>
+: <dfn>ad</dfn>
+:: A [=string=]. JSON string to be passed to the scoring function.
+: <dfn>ad descriptor</dfn>
+:: An [=ad descriptor=].
+: <dfn>ad component descriptors</dfn>
+:: Null or a [=list=] of [=ad descriptors=].
+: <dfn>ad cost</dfn>
+:: Null or a {{double}}. Advertiser click or conversion cost passed from generateBid() to
+  reportWin().
+: <dfn>modeling signals</dfn>
+:: Null or a non-negative 16-bit integer.

```

Comment:
Maybe just make this null or https://webidl.spec.whatwg.org/#idl-unsigned-short ?